tutorials/learning-api.conf.yaml

```diff
@@ -36,3 +36,15 @@
   category: Workflow example
   reading_time: 0
   catalog_featured: false
+
+- title: Compare transpiler settings
+  short_description: >
+    In this tutorial, we'll disable automatic transpilation and take you
+    through the full process of creating, transpiling, and submitting circuits.
```

Comment:
This is the old description, please update if necessary.

Reply:
Done!